### PR TITLE
Added package.json

### PR DESCRIPTION
**Pull request by theotherjimmy**
Auto-generated from the devjs-production-tools project

### CODE REVIEW

**Comment by theotherjimmy** [comment]
Found in `snap-pelion-edge`. Seems like something we should take upstream. Thanks @mray190.

**Comment by theotherjimmy** [comment]
@ygoyal18 Could you review this please?

File: package.json [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+    "json-formatter-js": "^2.2.0",
+    "jsonminify": "^0.2.3",
+    "jsonschema": "1.1.1",
+    "jsonwebtoken": "^7.4.3",
+    "md5": "^2.2.1",
+    "mkdirp": "^0.5.1",
+    "moment": "~2.10.6",
+    "node-array": "0.0.7",
+    "node-expat": "^2.3.18",
+    "node-znp": "git+ssh://git@github.com:armPelionEdge/node-znp.git#master",
```
Comment (whygoyal) [inline package.json]: @theotherjimmy Let's change this to https as this repo is public.

**Comment by whygoyal** [comment]
Looks good.

**Review by whygoyal — APPROVED** [review]

**Pull request merged**